utils/utils.go

```diff
@@ -24,12 +25,29 @@ func Min(a, b int) int {
 	return b
 }
 
+func runProviders(url string) error {
```

Comment:
Thanks for the PR!
Looks good, but can you rename this method? Providers is a kind of general term.
```suggestion
func openInBrowser(url string) error {
```